angular/src/scss/bwicons/styles/style.scss

```diff
@@ -4,8 +4,8 @@ $icomoon-font-path: "fonts" !default;
-    format("truetype");
+  src: url("../fonts/bwi-font.svg") format("svg"), url("../fonts/bwi-font.ttf") format("truetype"),
```

Comment:
That path needs to be dynamic. I.e. prefix it with a variable that is overriden in each client.